content/departments/engineering/teams/repo-management/index.md

```diff
@@ -33,6 +33,59 @@ Including maintenance, development, support, and advice for:
+
+1. Pageable alerts
+2. Active incidents
+3. Customer p0s and p1s
+4. Sentry alerts
+5. Customer p2s
+6. Monitor / action / resolve errors in #repo-management-errors channel on Slack
+7. Questions in #repo-management and #ask-engineering
+8. Incident action items (if not prioritized during planning)
+9. Tech debt
+
+We have the following expectations of the oncall:
+
+1. Triaging and resolving support requests is your only priority for the week. Any and all project work is put on hold for the week. When in doubt, reference the priorities above.
+2. Monitor #repo-management and #ask-engineering for inbound requests.
+3. Monitor any code monitors that post to our Slack channels
+4. Aim to acknowledge all requests within 24 hours, even if the initial response is an indication that we don’t have bandwidth to review it yet and will respond back at a future date. Add the GitHub issue to the Repo Management project with the Status “Support Issues”.
+5. Provide the incoming support owner with the status of all in-flight issues before their rotation begins (Monday morning is fine, asynchronous updates are fine). Include the relative priority of all issues so the incoming support owner knows where to begin.
+6. Ask for help when you are stuck! Don’t spend too much time trying to troubleshoot an issue, especially if it’s high priority. Do your best, ask the team, and resolve the issue.
+7. If you ever have too many support requests, leave a message in #repo-management-internal and engage the secondary support owner for the week
+8. After resolving an issue, do at least one of the following:
+
+- Add an FAQ to one of the existing docs based on the area or create one here
+- Update docs.sourcegraph.com or other documentation
+- Create a backlog issue for an improvement, feature, tool, etc. that would have been useful to troubleshoot and solve the problem and find an owner for the issue
+
+### Pageable Issues
+
+We have two ways of receiving pages. The first is for automated alerts that are defined in the next section for our Cloud customers. The second is when we are paged by other teams (typically the Cloud team) if a customer raises an issue with repo syncing. Below are our guidelines for manually paging our team.
+
+The following are pageable issues:
+
+1. 2 or more repos have not synced for more than 9 hours
+
+The following are not pageable issues:
+
+1. Partial or delayed repository sync that has occurred for less than 9 hours
+2. Repo syncing issues that occur during the 7 day instance creation window
+3. 1 repository is not syncing
+
+We ask that you send us a message in #repo-management for not pageable issues so we can help you.
```

Comment:
```suggestion
We ask that you send us a message in [#repo-management](https://sourcegraph.slack.com/archives/C02EDAQAJQZ) for not pageable issues so we can help you.
```

Reply:
The handbook is magic and will automatically link to the Slack channel 🙂

Reply:
🤯